index.js

```diff
@@ -12,3 +12,13 @@ I need this code, just don't know where, perhaps should make some middleware, do
 
 Go code!
 */
+
+
+const server = require('./server.js');
+
+const PORT = 6660;
```

Comment:
you'll want to get used to making this able to be deployed with little to no modifications so when we declare a port give it the const port = process.env.PORT || 6660; and you will have adefault and the env settings already done